README.md

```diff
@@ -1,2 +1,141 @@
 # Hello Teresa
 A collection of `Hello World` ready to deploy on [teresa](https://github.com/luizalabs/teresa-api) PaaS.
+
+## Deploy
+
+Some information to up and run your application on Kubernetes with Teresa.
```

Comment:
I'd write: `Some information to get your application up and running on Kubernetes with Teresa`